.github/workflows/get-images-scan-and-upload-report.yaml

```diff
@@ -0,0 +1,149 @@
+name: Get and scan images, and report vulnerabilities
+on:
+  workflow_call:
+    inputs:
+      bundle-path:
+        description: The bundle path on which the vulnerability scan will be executed
+        required: true
+        type: string
+      scans-out-dir:
+        description: The directory to store all vulnerability reports
+        required: false
+        type: string
+        default: "/tmp/trivy-reports"
```

Comment:
Good catch, I replaced all the instances where I hardcoded the `/tmp/trivy-reports` with `${{ inputs.scans-out-dir}}`.

Reply:
Probably not for users, but in case we want to re-use some of this code in another wf. If it does not make sense, I can remove it and hardcode the path.

Reply:
I 'm +1 to removing it. If you 'd like to keep this as a constant (to facilitate the reuse of the code), we could maybe have an envvar in `generate-and-upload-summary` job, since that's only where it's being used.